server/package.json

```diff
@@ -5,17 +5,18 @@
   "main": "server.js",
   "dependencies": {
     "body-parser": "^1.18.3",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v6.1.0",
```

Comment:
Note to self: update to real version number after merging https://github.com/bravetechnologycoop/brave-alert-lib/pull/32

Reply:
Done.